index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="node_modules/bootstrap/dist/css/bootstrap.min.css">
+    <title>NexNet | Login</title>
+</head>
+<body>
+    <main>
+        <section class="container auth-main">
+            <div class="row">
+                <div class="col-sm-12 col-md-6">
+                    <div>
+                        <h1>Welcome to NexNet</h1>
+                        <p>Connecting you to the world seamlessly.</p>
+                    </div>
+                    <div>
+                        <p>Need Help? Contact our support team at support@nexnet.com</p>
+                    </div>
+                </div>
+                <div class="col-sm-12 col-md-6">
+                    <form action="feed/" class="row">
+                        <div class="row-md-6 p-1">
+                            <input type="text" class="form-control" placeholder="Username">
+                        </div>
+                        <div class="row-md-6 p-1">
+                            <input type="password" class="form-control" placeholder="Password">
```

Comment:
Maybe you should ad HTML form validation here width method="post" in the form tag, plus minlength="8" and required on both of the inputs in the form.